services/queue/src/data.js

```diff
@@ -364,7 +364,7 @@ class TaskQueue {
 
   // Get a worker type from the DB, or undefined
   static async get(db, taskQueueId, expires) {
-    return TaskQueue.fromDbRows(await db.fns.get_task_queue(taskQueueId, expires));
+    return TaskQueue.fromDbRows(await db.deprecatedFns.get_task_queue(taskQueueId, expires));
```

Comment:
Good point. I thought that the queue service's APIs that used these methods were all deprecated. I was wrong. `getTaskQueue` and `listTaskQueues` are still stable.

I can't just simply switch over to the new functions that replaced these deprecated ones because they're for the worker manager service now. I'll need to continue work on this PR to replace these two methods with API implementations in worker manager.

Reply:
I didn't know this even exists :) `db.deprecatedFns` 
Usually when we deprecate we have a replacement right away, so probably it is better to adjust code to use new ones.. or keep old ones "un-deprecated"